app/config/config_test.yml

```diff
@@ -23,7 +23,8 @@ doctrine:
             default:
                 driver:   pdo_sqlite
                 path:     %kernel.cache_dir%/test.db
-
+                logging: false
```

Comment:
Now that we're not running mesh tests on travis can we keep logging on?  Its nice to be able to look at logs when tests fail.  Obviously we can switch it back on again, but its hard to remember where to do that.